orchestra/models.py

```diff
@@ -1,19 +1,74 @@
-from datetime import datetime
 from django.contrib.auth.models import User
 from django.db import models
```

Comment:
re: `I followed the existing convention I saw in other docstrings. jsonfield is probably better, unless it should be a dict (what you actually pass in to the field).

Happy to add an issue to clean it up everywhere if we decide on the right type.`

I think it should either be `dict` or `JSONField`.  Add an issue I guess?  I also feel like if you agree, we should set all of the stuff in this PR to be correct by our new standard.